.i18nrc.json

```diff
@@ -22,10 +22,8 @@
     "interpreter": "src/legacy/core_plugins/interpreter",
     "kbn": "src/legacy/core_plugins/kibana",
     "kbnDocViews": "src/legacy/core_plugins/kbn_doc_views",
-    "management": [
-      "src/legacy/core_plugins/management",
-      "src/plugins/management"
-    ],
+    "lists": "packages/kbn-securitysolution-list-utils/src",
+    "management": ["src/legacy/core_plugins/management", "src/plugins/management"],
```

Comment:
Ping-pong on the auto-formatter...

Reply:
Thanks, this change is so large I am going to leave it. Wish my editor hadn't done this.